apps/backend/src/users/types.ts

```diff
@@ -0,0 +1,31 @@
+// TODO: Probably want these types to be available to both the frontend and backend in a "common" folder
+export enum Status {
+  MEMBER = 'Member',
+  RECRUITER = 'Recruiter',
+  ADMIN = 'Admin',
+  ALUMNI = 'Alumni',
+  APPLICANT = 'Applicant',
+}
+
+export enum Team {
```

Comment:
:thought_balloon: should we add `EBOARD` as one of the possible values for this enum since the roles also contain all of the eboard positions? I think this will depend on what the purpose of the `team` property is (which we should follow up with product about)

Reply:
Im not too familiar with how the org's structure should be represented here but this does sound sensible. I think once we get clearer specifications, we can make a quick change here

Reply:
I added `EBOARD` to the types. We can see what the purpose of the `team` property is and remove it if necessary